app/src/main/AndroidManifest.xml

```diff
@@ -11,15 +11,25 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.DAZNMoviePlayer"
         tools:targetApi="31">
+        <activity
+            android:name=".SplashScreen"
+            android:exported="true"
+            android:theme="@style/Theme.SplashScreen"
+            tools:ignore="WrongManifestParent">
+        <intent-filter>
+            <action android:name="android.intent.action.MAIN" />
+
+            <category android:name="android.intent.category.LAUNCHER" />
+        </intent-filter>
+        </activity>
+
         <activity
```

Comment:
Can remove the activity code if not available

Reply:
remove unused tags